assets/app/styles/_core.less

```diff
@@ -404,6 +413,14 @@ code {
   white-space: normal;
 }
 
+pre {
+  &.code {
```

Comment:
are we sure we want to override the code style for the whole console?  assuming this means we are changing what we get from patternfly (bootstrap?)  cc @sg00dwin